root/static/scripts/common/MB/Control/Autocomplete.js

```diff
@@ -779,7 +779,7 @@ MB.Control.autocomplete_formatters = {
         if (item.typeName || (item.containment && item.containment.length)) {
             var items = [];
             if (item.typeName) {
-                items.push(_.escape(item.typeName));
+                items.push(item.typeName);
```

Comment:
This was not the visible issue; but if there are (translated) area type names with special characters, they would have been encode here and again on line 786, below.